nemo/export/trt_llm/converter/model_converter.py

```diff
@@ -203,7 +229,7 @@ def model_to_trtllm_ckpt(
         return weights_dicts, model_configs
 
     pp_key = {
-        "transformer.vocab_embedding.weight",
+        vocab_embedding_key,
```

Comment:
is this because the name of the embedding weight changes between TE FP8 and mcore? Or just to make the code cleaner?

Reply:
Just to make it cleaner